Changelog documentation (#180)
SolarArbiter/solarforecastarbiter_dashboard: CHANGELOG.md

# Changelog

> All notable changes to the Solar Forecast Arbiter Dashboard will be documented

Reviewer: This implies to me that this just tracks the Dashboard, but it's really anything that a dashboard user would encounter.

We should also encourage users to read the [`solarforecastarbiter-core` what's new](https://solarforecastarbiter-core.readthedocs.io/en/latest/whatsnew/index.html) series.

in this file.

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),.

## [Unreleased]

### Added

- Report form now includes inputs to select which metrics and categories to
  include in a Report.

- Ability to compare Forecasts to Aggregates during Report creation.

- Report signatures and HTML downloads for reports. Available at
  `/reports/<report_id>/downloads/html`. Note that there are no currently
  functioning links to this endpoint.

- CHANGELOG.md (this file) for tracking and communicating changes.

### Fixed

- Permissions acting on aggregates are now accessible via a Role's permission
  listing.

## [1.0beta2] - 2019-11-18

### Added
- Aggregates can be created through the dashboard. See  [Aggregate Documentation](https://solarforecastarbiter.org/documentation/dashboard/working-with-data/#create-new-aggregate) 

> 

Reviewer 2: ```suggestion

- Day-ahead probabilistic reference forecasts based on the GEFS are available for DOE RTC, NOAA SURFRAD, NOAA SOLRAD, and NREL MIDC networks.
```

# Changelog

All notable changes to the Solar Forecast Arbiter Framework will be documented
in this file.

Detailed changes to the Solar Forecast Arbiter Core python library can be found
in the core documentation's [what's new](https://solarforecastarbiter-core.readthedocs.io/en/latest/whatsnew/index.html) series.

> ## [Unreleased]

Reviewer 2: might as well change to 1.0beta3 and today's date

### Added

- Report form now includes all deterministic metrics options identified by

>   stakeholders. Also includes options to calculate metrics by categories Total,

Reviewer: Split into another `-` item. "Report form now includes options..."

  Year, Month, Date, and Hour of Day.

- Ability to compare Forecasts to Aggregates in reports.

> - Report's may be downloaded in HTML format at

Reviewer: ```suggestion
- Reports may be downloaded in HTML format at
```

  `/reports/<report_id>/downloads/html`. Note that there are no currently

>   functioning links to this endpoint.

Reviewer 2: There is a link in newer reports.

>   functioning links to this endpoint.

Reviewer: I don't understand the last sentence

- Report downloads contain a GPG signed report as well as md5, sha1 and sha256
  checksums for validation.

- CHANGELOG.md (this file) for tracking and communicating changes.

> 

Reviewer 2: is https://github.com/SolarArbiter/solarforecastarbiter-api/commit/8f66fe45beda692dd4d4d92bf82c6f5b14173051 worth adding?

Reviewer: sure

# Changelog

All notable changes to the Solar Forecast Arbiter Framework will be documented
in this file.

Detailed changes to the Solar Forecast Arbiter Core python library can be found
in the core documentation's [what's new](https://solarforecastarbiter-core.readthedocs.io/en/latest/whatsnew/index.html) series.

## [1.0beta3] - 2019-12-16

### Added

- Report form now includes all deterministic metrics options identified by
  stakeholders.

- Report form now includes options to calculate metrics by categories Total,
  Year, Month, Date, and Hour of Day.

> - Ability to compare Forecasts to Aggregates in reports.

Reviewer: ```suggestion
- Ability to analyze forecasts of aggregated observations in reports.
```

Reviewer: previous text assumed too much familiarity with data model
